streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -198,7 +198,7 @@ class StreamingContext private[streaming] (
 
   private var state: StreamingContextState = INITIALIZED
 
-  private val startSite = new AtomicReference[CallSite](null)
+  private[streaming] val startSite = new AtomicReference[CallSite](null)
```

Comment:
Looks it's better to add a new method like
```
private[streaming] def getStartSite: String = startSite.get()
``` 
rather than exposing `AtomicReference`. In addition, this one won't break MiMa tests.